inst/rmd/h/default.html

```diff
@@ -60,10 +67,15 @@
   overflow-x: auto;
 }
 .code-folding-btn {
-  margin-right: -30px;
+  margin-right: 0px;
+}
```

Comment:
This was introduced by #1399, and can be safely removed now.

Reply:
I got rid of this rule.